projects/kit/components/tabs/tab/tab.style.less

```diff
@@ -139,6 +139,7 @@
     width: 100%;
     align-items: center;
     justify-content: inherit;
+    text-align: left;
```

Comment:
Why do we need it?

Reply:
without

<img width="280" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/ca1dff2f-ee49-4be8-b545-ba1047f8990d">

Reply:
Weird :)